api/controllers/ExportController.js

```diff
@@ -161,8 +214,9 @@ function renderSkill(s) {
   return s.get('name')
 }
 
-function renderJoinRequestAnswer(s) {
-  return `${s[0].get('text')}: ${s[1].get('answer')}`
+function renderJoinRequestAnswersToJSON(QApair) {
+  if (QApair.length === 0) {return []}
```

Comment:
since we are looking for QApair[1] below shouldnt this ensure that length > 1?

Reply:
If there is a record, it will arrive in this function as an array of a question and answer. If there is no record, it will be an empty array. There isn't another eventuality here, so I feel it is fairly moot either way